packages/appsync-modelgen-plugin/src/__tests__/visitors/__snapshots__/appsync-model-introspection-visitor.test.ts.snap

```diff
@@ -1,5 +1,1529 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Custom primary key tests should generate correct model intropection file validated by JSON schema and not throw error when custom PK is disabled 1`] = `
```

Comment:
I don't think such a big snapshot makes sense to be added and is feasible to review. Please come up with an alternative test.